starter_code/public/javascript/APIHandler.js

```diff
@@ -4,22 +4,139 @@ class APIHandler {
   }
```

Comment:
You could have created a Function that calls to the API and then overwrite the arguments to call each new CRUD function...like this:
```
callToAPI (url, method, callback, data) {
    url = `${this.BASE_URL}${url}`;
    $.ajax({
      method: method,
      url: url,
      data: data,
      success: callback,
      error: this.errorHandler,
      datatype: "json",
    })
  }
```